### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by Kihau**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
 # Astar Java
-An implementation of an A* algorithm in Java
+Implementation of an A* algorithm in Java
```
Comment (FriskIsGit) [inline README.md]: Replace an with the

File: README.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# Astar Java
+An implementation of an A* algorithm in Java
+## [More information here]
```
Comment (FriskIsGit) [inline README.md:3]: *_*

File: README.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
 # Astar Java
-Implementation of an A* algorithm in Java
+Implementation of the A* algorithm in Java
```
Comment (FriskIsGit) [inline README.md]: Remov the entirely

**Review by FriskIsGit — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (Kihau) [reply README.md]: lol

**Review by FriskIsGit — APPROVED** [review]
LGTM

**Comment by Kihau** [comment]
Just merge it already pls

**Pull request merged**